### PR TITLE
Add "research" site and Researcher account

### PR DESCRIPTION
**Pull request by madprime**

### CODE REVIEW

File: open_humans/settings.py [hunk 1]
```diff
@@ -274,6 +278,14 @@ def __unicode__(self):
     }
 elif dj_database_url.config():
     DATABASES['default'] = dj_database_url.config()
+else:
+    # Default to sqlite if DATABASE_URL not set.
```
Comment (beaugunderson) [inline open_humans/settings.py]: does this actually work?

I think this will error out if you run `./manage.py syncdb; ./manage.py migrate` right?

**Comment by madprime** [comment]
oops that was a relic typo when I was messing around with things!

File: open_humans/settings.py (hunk 1 above)
Comment (madprime) [inline open_humans/settings.py]: Oops - this is a relic typo of me messing around with stuff! Need to fix.

**Comment by beaugunderson** [comment]
:+1: looks great except for the sqlite default; good to merge after that!

**Comment by beaugunderson** [comment]
awesome, I'll get to work making this show up on staging and get my study branch ready for merge! :)

---

**Comment by madprime** [comment]
awesome, nice to be making some concrete progress :)

**Pull request merged**